.github/workflows/trivy.yml

```diff
@@ -74,7 +74,7 @@ jobs:
         uses: aquasecurity/trivy-action@master
         with:
           # Path to Docker image
-          image-ref: "ghcr.io/catenax-ng/tx-autosetup-backend/autosetup:latest"
+          image-ref: "ghcr.io/catenax-ng/tx-managed-service-orchestrator/autosetup:latest"
```

Comment:
If you already come across this spot, please change the scan target to the images release via Tractus-X. Scan results on forks do not provide any value to the upstream, leading repo

Reply:
I did the changes in PR #58